packages/in_app_purchase/in_app_purchase_android/lib/src/in_app_purchase_android_platform_addition.dart

```diff
@@ -157,4 +157,26 @@ class InAppPurchaseAndroidPlatformAddition
         .runWithClient((BillingClient client) => client.getBillingConfig());
     return billingConfig.countryCode;
   }
+
+  /// Returns if the caller can use alternative billing only without giving the
+  /// user a choice to use Play billing.
```

Comment:
Do we expect to also need to support the with-user-choice version, or are we waiting to see if it's frequently requested first? I'm wondering if we should have an issue reference or anything here.

Reply:
Yes that is the next pr :) 
https://github.com/flutter/flutter/issues/143004

My intention is to expose all the newly added functionality in 6.1 this month.